samples/server/petstore/rust-server/output/openapi-v3/src/models.rs

```diff
@@ -629,16 +629,16 @@ impl AnyOfObject {
 #[cfg_attr(feature = "conversion", derive(frunk_enum_derive::LabelledGenericEnum))]
 pub enum AnyOfObjectAnyOf {
     #[serde(rename = "FOO")]
-    FOO,
+    Foo,
```

Comment:
@wing328 

>  sounds like that's a breaking change (not backward-compatible) right?

Yes that is correct, as you can see enum variants have changed to camel case in the rust server to comply with the Rust guidelines https://rust-lang.github.io/api-guidelines/naming.html

For example:

```
enum ExampleEnum {
    VARIANT_ONE
}
```

If we don't make this change then warnings are emitted:

```
warning: variant `VARIANT_ONE` should have an upper camel case name
  --> reqwest\petstore\src\lib.rs:13:5
   |
13 |     VARIANT_ONE
   |     ^^^^^^^^^^^ help: convert the identifier to upper camel case: `VariantOne`
   |
   = note: `#[warn(non_camel_case_types)]` on by default
```

Reply:
Edit: I can see that `rust-server` is supressing these warnings in lib.rs

Reply:
if the client prefers the old way of naming the enum, we will add an option for fallback later. Let's see if there's any demand for this from rust users after the v6.1.0 release.

Reply:
thanks, if this does come up - just tag me and I can write the code for you